packages/validator/templates/configSecurity.json

```diff
@@ -0,0 +1,98 @@
+{
+  "config": {
+    "ois": {
+      "__arrayItem": {
+        "__oisSpecs": {}
+      }
+    },
+    "triggers": {
+      "__optional": {
```

Comment:
Why are triggers optional? @bbenligiray

Reply:
The whole format is pretty redundant so `triggers.request` should be required here for consistency.

Reply:
Added to [template issue](https://github.com/api3dao/airnode/issues/197)